documentation/book/proc-renewing-ca-certs-manually.adoc

```diff
@@ -0,0 +1,51 @@
+// Module included in the following assemblies:
+//
+// assembly-security.adoc
+
+[id='proc-renewing-ca-certs-manually-{context}']
+
+= Renewing CA certificates manually
+
+Unless the `Kafka.spec.clusterCa` and `Kafka.spec.clientsCa` objects are set to `false`, the cluster and clients CA certificates will auto-renew at the start of their respective certificate renewal periods. 
+You can manually renew one or both of these certificates before the certificate renewal period starts, if required for security reasons. 
+A renewed certificate uses the same private key as the old certificate.
+
+.Prerequisites
+
+* The Cluster Operator is running.
+* A Kafka cluster in which CA certificates and private keys are installed.
+* One or more maintenance time windows are configured for the Kafka cluster.
+
+.Procedure
+
+* Apply the `strimzi.io/force-renew` annotation to the `Secret` that contains the CA certificate that you want to renew.
++
+[cols="3*",options="header",stripes="none",separator=¦]
+|===
+
+¦Certificate
+¦Secret
+¦Annotate command
+
+¦Cluster CA
+¦_<cluster-name>_-cluster-ca-cert
+m¦kubectl annotate secret _<cluster-name>_-cluster-ca-cert strimzi.io/force-renew=true
+
+¦Clients CA
+¦_<cluster-name>_-clients-ca-cert
+m¦kubectl annotate secret _<cluster-name>_-clients-ca-cert strimzi.io/force-renew=true
+
+|===
+
+If a maintenance time window is active, at the next reconciliation the Cluster Operator generates a new CA certificate for the `Secret` that you annotated. 
```

Comment:
This is good. But I think I would add one more sentence at the end of this paragraph. Something like _If you are not using the maintenance time window, the new CA certificate will be generated in the next reconciliation._ since right now it could be confusing for users who do not use the maintenance time window at all.

Reply:
Agree. See my comment above about "Prerequisites". Havina "one or more" maintenance time window is not a prerequisite.